drivers/bridge/bridge.go

```diff
@@ -218,14 +225,24 @@ func (c *networkConfiguration) fromLabels(labels map[string]string) error {
 			if c.EnableICC, err = strconv.ParseBool(value); err != nil {
 				return parseErr(label, value, err.Error())
 			}
-		case DefaultBridge:
-			if c.DefaultBridge, err = strconv.ParseBool(value); err != nil {
-				return parseErr(label, value, err.Error())
-			}
 		case DefaultBindingIP:
 			if c.DefaultBindingIP = net.ParseIP(value); c.DefaultBindingIP == nil {
 				return parseErr(label, value, "nil ip")
 			}
+		case DefaultBridgeLegacyIPv6Network:
```

Comment:
That would work, but decision whether to operate that way (allocating the address pre or post CreateEndpoint) should not be taken by libnetwork alone and based on mask lengths. The address allocation strategy should be controlled at least by a network driver property. 
Also we probably want this behavior (deterministic ipv6 address setting implicitly via --mac-address run option) be preserved only for default bridge network, not for all bridge networks.